### DISCUSSION TITLE
Treetable with checkbox selection feature request

### DISCUSSION THREAD

**Discussion by csakis** [primefaces/community#929] [PrimeVue] [answered]
I have a tree table with selectionMode "checkbox" enabled.  I need to disable the checkboxes conditionally at the leaf level in the tree. If I use the template `<template #checkboxicon>`  in the TreeTable component, I am able to disbale the checkboxes for the whole tree, but using   `<template #rowcheckboxicon>` on the Columns does not seem to work.
Further more, I am able to template the icons themselves, but not the whole checkbox component of the tree. 
Am I missing something? Is there an API, passthrough option or template that I can use to achive conditional disabling checkboxes in TreeTable rows?

Here's a Stackblitz example:
https://stackblitz.com/edit/phshwl?file=src%2FApp.vue

**Comment by csakiss26** [comment]
Has anyone from PrimeTek read this? Is it something they consider or I'm on my own with this issue?

**Comment by tugcekucukoglu** [comment, marked as answer]
`rowcheckbox` option of `pt` property can disabled the checkboxes.

Example: https://stackblitz.com/edit/xe3hso?file=src%2FApp.vue
